README.md

```diff
@@ -150,6 +150,7 @@ Launches the test runner in the interactive watch mode.
 👤 **Author**
 
 - GitHub: [@tjay1760](https://github.com/tjay1760)
+- GitHub: [@rbd3](https://github.com/rbd3)
 
```

Comment:
- [ ] @Team it would be nice if you added more links to your social accounts eg Twitter and LinkedIn, I also suggest you separate the links of the authors so that each can be under an `👤 Author` header.